### ISSUE TITLE
zhoubear/open-paperless

### ISSUE THREAD

**Issue by pawamoy** [the-paperless-project/paperless#507]
I'm reviewing my GitHub starred list. Just saw that: [zhoubear/open-paperless](https://github.com/zhoubear/open-paperless). What bothers me a bit is that both `the-paperless-project` and `zhoubear`
 projects use the same description line: `Scan, index, and archive all of your paper documents` (with `(acquired by Mayan EDMS)` as a bonus in `zhoubear`'s one).

What's happening here? (I'm just curious, if anyone has more info)

**Comment by syntonym** [comment]
`zhoubear/open-paperless` is a mayan EDMS fork which focusses on simplifying the mayan EDMS user interface. `the-paperless-project/paperless` has no relationship to mayan EDMS (or zhoubear/open-paperless) as far as I know. In `the-paperless-project/paperless` the second commit introduced the description as `Scan, index, and archive all of your paper bills` which was then changed to the current description one commit later (cbc8c25). These commits were done in december 2015 by Daniel Quinn (original author and still active), while the first commit of zhoubear/open-paperless is in 2017.

I'd guess the matching description is just a coincidence, as it's a pretty clear description of the goal of these both projects.

**Comment by danielquinn** [comment]
That project has a rather interesting history actually.  There was some rather heated discussion on Reddit back when it first was linked there.  As @syntonym said, it's a re-skinning on Mayan EDMS, but critically back when the project first came out, *no mention of Mayan was made*.  As I understand it, credit was even scrubbed from the codebase.

This was all met by some angry people shouting down @zhoubear for her work.  If I recall correctly, she explained that she was just trying to do the right thing: repurpose one project and build on top of it, but that her way of doing so (not properly crediting the original authors) wasn't done with any malice intended.  She later addressed this by restoring the copyright & credit info, so as I understand it, it's a non-issue now.

As for the name and tag line "scan, index, and archive all of your paper" being identical to this one, I was a little freaked out when I first saw that (especially as it implied that Paperless was somehow *not* open), but I have since decided to consider it flattery :-)

That Reddit thread appears to have gone away since, which is too bad, 'cause there were a bunch of people spinning conspiracy theories about how this project is a total rip off of Mayan and how I'm some sort of Big Bad Interloper who stole all that hard work... and yet somehow totally rewrote it all... I think.  The logic there was unclear.  Good times.

Regardless, I [met](https://danielquinn.org/blog/djangocon-2017/) Roberto Rosario (the Mayan EDMS author) at DjangoCon back in 2017 and he assured me that there's no grumpiness coming from his end.  He's a pretty cool guy actually.

I'm gonna close this, but if you've got more questions, just re-open and I'll try to respond when I've got time.
